dev/com.ibm.ws.concurrent.persistent_fat_configupdate/publish/servers/com.ibm.ws.concurrent.persistent.fat.configupdate/server.xml

```diff
@@ -0,0 +1,53 @@
+<!--
+    Copyright (c) 2019 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+   
+    Contributors:
+        IBM Corporation - initial API and implementation
+ -->
+<server>
+  <featureManager>
+    <feature>concurrent-1.0</feature>
+    <feature>jndi-1.0</feature>
+    <!-- <feature>osgiConsole-1.0</feature> -->
+    <feature>persistentExecutor-1.0</feature>
+    <feature>servlet-3.1</feature>
+    <feature>jdbc-4.2</feature>
```

Comment:
it looks like this bucket is set up to run on any Java level, including java 7, but jdbc-4.2 only works for Java 8+. I believe it is valid for these tests to be running on Java 7, so lets downgrade to jdbc-4.1 here.